components/form/form-element-mixin.js

```diff
@@ -171,6 +185,9 @@ export const FormElementMixin = superclass => class extends LocalizeCoreElement(
 				this.dispatchEvent(new CustomEvent('invalid-change'));
 			}
 		}
+		if (this.validateOnInit && (changedProperties.has('noValidate') || changedProperties.has('validateOnInit'))) {
```

Comment:
Why does `noValidate` changing affect this?

Reply:
So if the `noValidate` attribute is removed later, it will immediately revalidate.

Reply:
Gotcha.